sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample10_AzureEventSourceListener.md

```diff
@@ -127,7 +127,7 @@ finally
 
 ## Capture filtered logs to a file
 
-For scenarios where capturing logs to `Trace` or `Console` outputs isn't ideal, log information can be streamed into a variety of targets, such as Azure Storage, databases, and files for durable persistence.    This example demonstrates capturing error logs to a text file so that they can be analyzed later, while capturing non-error information to `Console` output.  
+For scenarios where capturing logs to `Trace` or `Console` outputs isn't ideal, log information can be streamed into a variety of targets, such as Azure Storage, databases, and files for durable persistence.    This example demonstrates capturing error logs to a text file so that they can be analyzed later, while capturing non-error information to `Console` output.
```

Comment:
```suggestion
For scenarios where capturing logs to `Trace` or `Console` outputs isn't ideal, log information can be streamed into a variety of targets, such as Azure Storage, databases, and files for durable persistence.  This example demonstrates capturing error logs to a text file so that they can be analyzed later, while capturing non-error information to `Console` output.
```

Reply:
I was trying to keep this PR focused on just this change. If you want me to go through and fix up a lot of the markdown problems as found by the Microsoft Docs extension pack, it may take several days. If that's worthwhile, can we do that in another PR?

Also, the double-space in the second recommendation - the archaic typewriter style - will actually be folded into a single space by markdown renderers. Same with the trailing space, though those are not recommended by git. It actually highlights them in an initial diff.

Reply:
Double-space 4Lyfe!

Reply:
As discussed offline, this isn't necessary now. Perhaps a separate pass to resolve markdown linting rules would help, but unless people actually have the Microsoft Docs extension pack, more issues will only be introduced later. I can mention it in my "PSA" talking about this change to the team.